scene/2d/gpu_particles_2d.cpp

```diff
@@ -41,6 +41,9 @@
 void GPUParticles2D::set_emitting(bool p_emitting) {
 	// Do not return even if `p_emitting == emitting` because `emitting` is just an approximation.
 
+	if (p_emitting && p_emitting != emitting && !use_fixed_seed) {
```

Comment:
```suggestion
	if (p_emitting && !emitting && !use_fixed_seed) {
```
Looks like it could be simplified, right? The same is valid below.